### PR TITLE
[frontend] Extract common styles

### PR DESCRIPTION
**Pull request by Goumies**
### Proposed changes

* Reduce duplication by extracting common styles
* This is the first step to refactoring and optimizing styles

### Checklist

- [x] I consider the submitted work as finished
- [ ] I tested the code for its functionality
- [ ] I wrote test cases for the relevant uses case
- [ ] I added/update the relevant documentation (either on github or on notion)
- [x] Where necessary I refactored code to improve the overall quality

### Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: opencti-platform/opencti-front/src/private/components/locations/regions/RegionCreation.tsx [hunk 1]
```diff
@@ -94,7 +41,7 @@ const regionMutation = graphql`
   }
 `;
 
-interface RegionAddInput {
+export interface RegionAddInput {
```
Comment (marieflorescontact) [inline opencti-platform/opencti-front/src/private/components/locations/regions/RegionCreation.tsx:44]: why the `export` ?

File: opencti-platform/opencti-front/src/private/components/locations/regions/RegionCreation.tsx [hunk 2]
```diff
@@ -113,7 +60,7 @@ interface RegionFormProps {
   inputValue?: string;
 }
 
-const REGION_TYPE = 'Region';
+export const REGION_TYPE = 'Region';
```
Comment (marieflorescontact) [inline opencti-platform/opencti-front/src/private/components/locations/regions/RegionCreation.tsx:63]: same `export` question ?

File: opencti-platform/opencti-front/src/private/components/common/style/commonStyle.ts [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+		minHeight: '100vh',
+		width: '50%',
+		position: 'fixed',
+		transition: theme.transitions.create('width', {
+			easing: theme.transitions.easing.sharp,
+			duration: theme.transitions.duration.enteringScreen,
+		}),
+		padding: 0,
+	},
+	dialogActions: {
```
Comment (marieflorescontact) [inline opencti-platform/opencti-front/src/private/components/common/style/commonStyle.ts:15]: seems to be unused in the components affected by refactoring

File: opencti-platform/opencti-front/src/private/components/common/style/commonStyle.ts [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+		backgroundColor: theme.palette.background.nav,
+		padding: '20px 20px 20px 60px',
+	},
+	closeButton: {
+		position: 'absolute',
+		top: 12,
+		left: 5,
+		color: 'inherit',
+	},
+	importButton: {
```
Comment (marieflorescontact) [inline opencti-platform/opencti-front/src/private/components/common/style/commonStyle.ts:46]: seems to be unused in the components affected by refactoring

**Comment by marieflorescontact** [comment]
we should discuss the appropriate strategy for mutualizing style with @RomuDeuxfois and @Kedae

File: opencti-platform/opencti-front/src/private/components/locations/regions/RegionCreation.tsx (hunk 1 above)
Comment (Goumies) [reply opencti-platform/opencti-front/src/private/components/locations/regions/RegionCreation.tsx:44]: That's a leftover from my attempt to generify the CustomFileUploader component.
I have suppressed it in the initial PR.

File: opencti-platform/opencti-front/src/private/components/locations/regions/RegionCreation.tsx (hunk 2 above)
Comment (Goumies) [reply opencti-platform/opencti-front/src/private/components/locations/regions/RegionCreation.tsx:63]: That's a leftover from my attempt to generify the CustomFileUploader component.
I have suppressed it in the initial PR.

**Pull request closed without merging**